profile/CSIP.xml

```diff
@@ -1224,7 +1224,8 @@
             <requirement ID="CSIP105" REQLEVEL="SHOULD" EXAMPLES="structMapExample2">
                 <description>
                     <head>Representation divisions</head>
-                    <p xmlns="http://www.w3.org/1999/xhtml">When the transfer consist of representations there are one representation div present for each representation</p>
+                    <p xmlns="http://www.w3.org/1999/xhtml">When the transfer consist of content divded into representations described by its own METS.xml documents there are one representation div present for each representation.</p>
+                    <p xmlns="http://www.w3.org/1999/xhtml">The different representation divisions gives pointers to the METS-document thus providing the structural map display of the whole package and its different representations.</p>
```

Comment:
Each representation div references the representation level METS.xml document, documenting the structure of the package and its constituent representations.